src/main/java/org/takes/facets/flash/RsFlash.java

```diff
@@ -117,7 +111,6 @@
     /**
      * To string.
      */
-    @SuppressWarnings("unused")
     private final transient CharSequence text;
```

Comment:
@dalifreire strange, so what does this class do? Nothing? Maybe their only responsibility is to provide constructors and actual behaviour is defined in parents